integration_tests/pages/manage/placementShow.ts

```diff
@@ -0,0 +1,36 @@
+import { Cas1PremisesSummary, Cas1SpaceBooking, FullPerson } from '@approved-premises/api'
+import Page from '../page'
+import paths from '../../../server/paths/manage'
+import { DateFormats } from '../../../server/utils/dateUtils'
+
+export default class PlacementShowPage extends Page {
+  constructor(placement: Cas1SpaceBooking) {
+    const pageHeading = `${DateFormats.isoDateToUIDate(placement.canonicalArrivalDate, { format: 'short' })} to ${DateFormats.isoDateToUIDate(placement.canonicalDepartureDate, { format: 'short' })}`
+    super(pageHeading)
+    this.checkPhaseBanner('Give us your feedback')
+  }
+
+  static visit(premisesId: string, placement: Cas1SpaceBooking): PlacementShowPage {
+    cy.visit(paths.premises.placements.show({ premisesId, placementId: placement.id }))
+    return new PlacementShowPage(placement)
+  }
+
+  shouldShowPersonHeader(placement: Cas1SpaceBooking): void {
+    const { name, crn, dateOfBirth } = placement.person as FullPerson
+    cy.get('.key-details-bar').should('contain', name)
+    cy.get('.key-details-bar').should('contain', crn)
+    cy.get('.key-details-bar').should('contain', DateFormats.isoDateToUIDate(dateOfBirth, { format: 'short' }))
+  }
+
+  shouldShowSummaryInformation(placement: Cas1SpaceBooking, premises: Cas1PremisesSummary): void {
+    ;[
+      ['Delius Event Number', placement.deliusEventNumber],
+      ['AP name', premises.name],
+      ['Key worker', placement.keyWorkerAllocation.keyWorker.name],
+      ['Expected arrival date', DateFormats.isoDateToUIDate(placement.expectedArrivalDate)],
+      ['Expected departure date', DateFormats.isoDateToUIDate(placement.expectedDepartureDate)],
+    ].forEach(([title, expectedText]) => {
+      cy.contains(title).closest('.govuk-summary-list__row').should('contain', expectedText)
+    })
+  }
```

Comment:
Now there's interesting! It feels wrong to use the same code that drives the template in an integration test, but I guess that's tested in the unit tests so it's OK and is a lot easier.

Reply:
Actually, it turned out to be a bit messy because that method doesn't handle html in both the key and value. I've split out the tests to cope and had to add a bodgy filter - but that can go when we remove the grey text.